scripts/oneoff/acknowledge_publish_draft_service_updates.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import argparse
+import multiprocessing as mp
+from multiprocessing.pool import ThreadPool
+import os
+import sys
+
+from dmapiclient.audit import AuditTypes
+from dmapiclient.data import DataAPIClient
+
+sys.path.insert(0, '.')  # noqa
+
+from dmscripts.helpers.auth_helpers import get_api_url, get_auth_token
+
+def acknowledge_update(audit_event):
+    response = data.acknowledge_audit_event(audit_event['id'], user='(dm developers) scripts/publish-g-cloud-draft-services.py')['auditEvents']
+    print(f'Acknowledged {audit_event["id"]} on service id #{audit_event["data"]["serviceId"]} by supplier `{audit_event["data"]["supplierName"]}`: {response["acknowledged"]}')
```

Comment:
Maybe add a check for a 200?

Reply:
API client doesn't return a status code, it raises errors on non-200.

Reply:
cool